pkg/virtctl/vmexport/vmexport.go

```diff
@@ -64,6 +64,7 @@ const (
 	INSECURE_FLAG = "--insecure"
 	KEEP_FLAG     = "--keep-vme"
 	PVC_FLAG      = "--pvc"
+	TTL_FLAG      = "--ttl"
```

Comment:
Is there a functional test for this?

Reply:
Nope, should I add one and drop the current functional test?
These tests are a little expensive and would basically duplicate each other

Reply:
the virtctl test can simply check that export resource is created with the correct params and exit.  The existing test can make sure everything works

Reply:
done, thx